File: Chapter 1/Problem 13/Ch1_Problem13.java

public class Ch1_Problem13 {
    public static void HanoiMoves(int n, char s, char d, char a) {
        if (n == 1) System.out.println(s + "->" + d);
        else{
            HanoiMoves((n - 1), s, a, d);
            System.out.println(s + "->" + d);
            HanoiMoves((n - 1), a, d, s);
        }
    }
}

File: Chapter 1/Problem 13/Ch1_Problem13.cpp
#include <iostream>
using namespace std;

void HanoiMoves(int n, char s, char d, char a)
{
    if (n == 1)
        printf("\n%c -> %c", s, d);
    else
    {
        HanoiMoves((n - 1), s, a, d);
        printf("\n%c -> %c", s, d);
        HanoiMoves((n - 1), a, d, s);
    }
}